wolfssl/wolfcrypt/settings.h

```diff
@@ -396,6 +399,30 @@
     #define WOLFSSL_ESP32
 #endif
 
+#if defined(NO_ESP32WROOM32_CRYPT)
+    #undef NO_ESP32WROOM32_CRYPT
+    #define NO_ESP32_CRYPT
+    #warning "Please use NO_ESP32_CRYPT not NO_ESP32WROOM32_CRYPT"
```

Comment:
Recommend using `#error`, not `#warning`.

Reply:
Done